README.md

```diff
@@ -1,75 +1,128 @@
-	5.	Get diesel backup generation data
-		a.	In your location folder, open the “Generation” folder and then the “Diesel” folder
-		b.	Complete the “Diesel generation inputs.csv” template with the details of your location
-	6.	Get load data
-		a.	In your location folder, open the “Load” folder
-		b.	Complete the “Devices.csv” template with the details of your location
-		c.	In the “Devices utilisation” folder, complete the utilisation profiles for each device e.g. “light_times.csv”
-			i.	Utilisation profiles are a 12x24 (monthly x hourly) matrix of probabilities that the specified device is in use in that hour
-			ii.	Each device in  “Devices.csv” must have a corresponding utilisation profile
-		d.	Run Load().number_of_devices_daily() to get the number of each device in the community on a given day
-		e.	Run Load().get_device_daily_profile() to get the daily utilisation profile (365x24 matrix) for each device
-		f.	Run Load().devices_in_use_hourly() to generate the number of devices in use for each hour
-		g.	Run Load().device_load_hourly() to get the load of each device
-		h.	Run Load().total_load_hourly() to get the total community load, segregated into “Domestic”, “Commercial” and “Public” demand types
-	7.	Set up the energy system
-		a.	In your location folder, open the “Simulation” folder
-		b.	Complete the “Energy system inputs.csv” template with the details of your location
-		c.	In your location folder, open the “Scenario” folder
-		d.	Complete the “Scenario inputs.csv” template with the details of your location
-	8.	Perform a simulation
-		a.	Run Energy_System().simulation(start_year, end_year, PV_size, storage_size) with your chosen system
-		b.	Record the outputs as a variable to investigate the outputs in more detail
-		c.	Save the outputs using Energy_System().save_simulation(simulation_name,filename)
-		d.	Open a saved simulation using Energy_System().open_simulation(filename)
-	9.	Input financial information
-		a. 	In your location folder, open the "Impact" folder
-		b. 	Complete the "Financial inputs.csv" with details of your location
-	10.	Input GHG information
-		a. 	In your location folder, open the "Impact" folder
-		b. 	Complete the "GHG inputs.csv" with details of your location
-	11.	Set up the optimisation process
-		a.	In your location folder, open the "Optimisation" folder
-		b.	Complete the “Optimisation inputs.csv” template with the details of your location
-	12.	Perform an optimisation
-		a. 	Run Optimisation().multiple_optimisation_step()
-		b.	Record the outputs as a variable to investigate the outputs in more detail
-		c.	Save the outputs using Optimisation().save_optimisation(optimisation_name,filename)
-		d.	Open a saved optimisation using Optimisation().open_optimisation(filename)
+
+CLOVER minigrid simulation and optimisation for supporting rural electrification in developing countries.
+
+## CLOVER Quick Start Guide
```

Comment:
Thanks for updating this, it wasn't on my radar but glad it was on yours!

Reply:
No worries! Thought that the first question (when I eventually manage to show it :p ) would be, "how do I run it?"